src/posts/blog/6-masks/masks.ts

```diff
@@ -0,0 +1,277 @@
+*   The nose bridge and the bottom of the chin are where fit is most often an issue.  Any good N95 will have a wire or metal strip on the nose bridge so that it will fit your nose.
+*   The mask should have straps that go around the back of your head, not just ear loops.  Head straps give you a tighter fit
+
+### Reusing your N95
+*   You’ll need to throw out your N95 mask if it gets wet, overly dirty, or difficult to breathe through.
+*   If you have a large supply of N95 masks and there are no shortages in your region, you should consider retiring each mask after 8 hours of total use.  This is the recommended lifetime stated by most manufacturers.  However, if supplies are low, it is possible to continue to use the same mask for longer than 8 houes.  While most N95 masks designed for 8 hours, and in high-particulate settings (e.g., wildfires) masks will become clogged during this time, it appears that most N95 masks can still achieve good performance over longer periods of use in everyday settings where the main goal is to filter viruses.  The [CDC specifically recommends this](https://www.cdc.gov/coronavirus/2019-ncov/hcp/ppe-strategy/decontamination-reuse-respirators.html#:~:text=The%20healthcare%20staff%20member%20can,off%E2%80%9D%20during%20storage.) when supplies are low, though it notes that filtration efficiency will decrease during extended use.  However, even an N95 that has been used many times is still likely better protection than a cloth mask.     
+*   Since you’ll likely reuse the mask, be sure to treat the outer surface as potentially contaminated after you wear it, especially if you’ve been to a high risk location.  A mask that’s been sitting unused for 8 days can be considered sterilized again as nearly all viruses decay by that time.  
+
+**Exhalation valves:** It should be noted that some N95s come with an exhalation valve.  While this makes the mask easier to breathe through, it lets you expel air from your mouth without any filtration, which isn’t safe for the people around you.  However, you don’t need to throw these masks out; you can just tape over the exhalation valve (on the outside, so other people know) and then wear it like a normal N95, or just put a surgical mask over it.
+
+<figure>
+<img src=${exhalationvalvetape} width="531", height="258"/>
+</figure>
+
+**How N95s work:** N95 (and KN95) masks work on a very different principle from cloth masks -- they have an electrostatic charge that attracts small airborne particles that pass through the mask.  Water destroys the electric charge, so getting the mask wet will make it stop working.  Separately, the mask will stop working effectively once too many particles have become caught in it.  Here’s a [cute video explanation](https://www.youtube.com/watch?v=eAdanPfQdCA).
```

Comment:
Still no source for wetting being a problem. Prefer "getting the mask wet may degrade its performance"